VogtFactory/Dodge.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtFactory
+{
+    class Dodge: RentalCar
+    {
+        public string Name = "minivan";
+        public int NumberOfPeopleThatCanFit = 7;
+        public Dodge()
```

Comment:
Can get rid of the constructor. It's not required. No reason to have it if its not doing anything.